src/ontology/lexmatch/README.md

```diff
@@ -3,48 +3,47 @@
 * mondo-only: Positive mappings in MONDO not caught by the lexical mapping pipeline
 * split-mapping-set: Unmapped mappings broken down by predicate_id
 ## Summary of mappings:
- * Number of mappings in [`unmapped_doid_lex`](unmapped_doid_lex.tsv): 218
- * Number of mappings in [`unmapped_doid_lex_exact`](unmapped_doid_lex.tsv): 132
+ * Number of mappings in [`unmapped_doid_lex`](unmapped_doid_lex.tsv): 29
+ * Number of mappings in [`unmapped_doid_lex_exact`](unmapped_doid_lex.tsv): 9
  * Number of mappings in [`unmapped_doid_mondo`](mondo-only/unmapped_doid_mondo.tsv): 38
  * Number of mappings in [`unmapped_doid_mondo_exact`](mondo-only/unmapped_doid_mondo.tsv): 38
- * Number of mappings in [`unmapped_icd10cm_lex`](unmapped_icd10cm_lex.tsv): 1934
- * Number of mappings in [`unmapped_icd10cm_lex_exact`](unmapped_icd10cm_lex.tsv): 1532
+ * Number of mappings in [`unmapped_icd10cm_lex`](unmapped_icd10cm_lex.tsv): 1956
+ * Number of mappings in [`unmapped_icd10cm_lex_exact`](unmapped_icd10cm_lex.tsv): 1553
```

Comment:
Are the number of mappings in `[unmapped_icd10cm_lex](unmapped_icd10cm_lex.tsv): 1956` compared to `[unmapped_icd10cm_lex](unmapped_icd10cm_lex.tsv): 1934` increasing because this source has not been reviewed vs. compared to doid where the number has decreased and DO was reviewed before this data release?

Reply:
Exactly! Both your suppositions are correct!